config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   root 'tasks#index'
+  post "tasks/task_complete"
```

Comment:
Don't define routes like this. If you have a resource defined, define all the routes for that resource in a block.